GlyssenEngineTests/Character/CharacterVerseDataTests.cs

```diff
@@ -41,9 +38,73 @@ public void IsCharacterStandard_StandardCharacter_ReturnsTrue(string characterId
 		[TestCase("narrator's friend")]
 		[TestCase("extraordinary dude in MAT")]
 		[TestCase("someone speaking in intro-REV")]
```

Comment:
These test cases were for normal characters aren't they?  They are not being used for non standard characters.  Is this correct?  This includes line 37 (Jesus).